### DISCUSSION TITLE
quantum computer for plot hashing

### DISCUSSION THREAD

**Discussion by ktkiebel** [Chia-Network/chia-blockchain#7767] [General]
if quantum computers become more mainstream and can hash chia plots quickly does it have the potential to reduce the chia blockchain value ?

**Comment by rmica** [comment]
Chia value is manly in space not in cpu power. You can plot faster, and you need more space. Maybe I'm wrong, but I can't see a value of processing power other than plotting faster.

**Comment by SlugFiller** [comment]
Problems solvable by quantum computers (BQP) are a subset of problems solvable by infinite parallelism (NP). Given that part of the plotting design principles is to prevent sufficiently-powerful parallelism from being profitable compared to simply having more storage, this is highly unlikely, barring a truly unexpected discovery in cryptography and computer algorithms.

Quantum computers are far more likely to be a risk to the asymmetric key encryption that encapsulates both the wallets, and the farmer key, than the plotting algorithm itself.

If anything, PoW, which relies primarily on parallelism, is at higher risk. However, the difference between BQP and NP is sufficiently large that this is not likely to become an issue in the foreseeable future.
